Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.11.11-slim
+LABEL maintainer="leon.kushnir15@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+CMD ["python", "app/main.py", "runserver", "0.0.0.0:8000"]

```

Comment:
The `CMD` instruction is attempting to run multiple commands (`app/main.py`, `runserver`, `0.0.0.0:8000`). The `CMD` instruction should only execute a single command. If you need to run multiple commands, consider using a shell script or chaining commands with `&&`.